src/main/java/org/symphonyoss/integration/webhook/salesforce/SalesforceWebHookIntegration.java

```diff
@@ -40,6 +44,10 @@
 @Component
 public class SalesforceWebHookIntegration extends WebHookIntegration {
 
+  private static final String typeJSON = "application/json";
```

Comment:
You don't need to declare this constant, you should use MediaType.APPLICATION_JSON.